tests/unit/htdocs/class/auth/AuthLdapEscapeTest.php

```
@@ -0,0 +1,59 @@
+        self::assertStringNotContainsString('*', $filter);
+        self::assertStringNotContainsString(')(', $filter);
+        self::assertStringContainsString('\2a', strtolower($filter));
+    }
+
+    #[Test]
+    public function userDnEscapesMetacharacters(): void
+    {
+        if (!function_exists('ldap_escape')) {
+            self::markTestSkipped('ext-ldap not available');
+        }
+        $dn = $this->ldap()->getUserDN('a,b+c');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $dn. Configured minimum length is 3. (undefined)

(ShortVariable)